### ISSUE TITLE
How to integrate APIs in this template

### ISSUE THREAD

**Issue by abdullahqureshi1994** [creativetimofficial/light-bootstrap-dashboard-react#56]
Can anyone please let me know how to integrate APIs using this template.

I have tried to integrate APIs using 'axios' in this template where APIs created in Node.js.
I got this error in the browser

Access to XMLHttpRequest at 'http://localhost:3000/user/import' from origin 'http://localhost:4000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

      `constructor(props) {
		super(props);
		this.state = {
			users: []
		};
		this.load_table();
	}

	componentDidMount() {
		this.load_table();
	}

	async load_table(){

		const instance = await axios.post('http://localhost:3000/user/import',
			{	headers: 
					{'x-access-token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVkNTY1YWUwYWE3NjNiNDlhNDY1ODQ1YyIsImlhdCI6MTU2NjA0ODA4MiwiZXhwIjoxNTY2MTM0NDgyfQ.aGX0AhwL-9DrZZ20F5vJqPnqqYRThtWlaDe_GfGW3kU'
					}
			}
			);
		console.log('instance>>',instance);
		
		this.setState({
			users: []
		});

	}

**Comment by EINazare** [comment]
Hello there, @abdullahqureshi1994 

Thank you for your interest in working with our products.
It seems that you haven't respected our ISSUE_TEMPLATE, so we are going to close it.
Please add a new comment here, or open a new issue, by filling the mentioned ISSUE_TEMPLATE and please be sure that your issue title is a very small description of your issue as well.

All the best,
Manu
Developer @[Creative-Tim.com](http://creative-tim.com/)
